native/v-kit.sh

```diff
@@ -419,6 +423,7 @@ main() {
     setup_systemd_skip
```

Comment:
Setting up skip sidecar requires root permissions: https://docs.skip.build/connect/validators/quickstart#using-curl. This might be an issue too.

Reply:
The install should still be done with root/sudo, and the curl | bash is performed at that time.